Trinket Foul fixed, Prot Warrior Mitigation fixed, and Prot Warrior unused setting removed.

Spike2D opened the pull request.

Spike2D committed:
Fixed Prot Warrior Mitigations

Spike2D committed:
Foul Trinket ID Fixed

Spike2D committed:
Removed last_stand_offensively

Spike2D committed:
Added updated profile

Spike2D committed:
Victory Rush settings added, SimC updated.

Spike2D committed:
Victory Rush added to SimC

Spike2D committed:
Formatting of Victory Rush Setting

Spike2D committed:
More fixed to Victory Rush

johnnylam88 commented:
The defensives should also have `use_off_gcd=1`.

johnnylam88 commented:
To mirror the wording used in the other settings, I would change the description on the Victory Rush setting from:

> When set above zero, the addon may recommend |T589768:0|t Victory Rush when your health falls below this percentage.

to:

> If set below 100, the priority can recommend |T589768:0|t Victory Rush if your current health has fallen below this percentage.

The nuance between saying the addon does it versus the priority is between whether the ability has a handler that checks that setting, or if the priority checks that setting.

Spike2D commented:
Awesome, will add those changes right away @johnnylam88 
Edit:
+ Added use_off_gcd on relevant skills
+ Modified settings to work more in line to the Prot Paladin changes, and fixed the wording on the settings.

Spike2D committed:
Add files via upload

Spike2D committed:
Add files via upload
